templates/layout.html

```diff
@@ -43,8 +43,10 @@
         <div class="col-md-12">
           <div class="menu btn-group">
             <a class="btn btn-link logo" href="/">cdnjs</a>
+
```

Comment:
all the styling changes and not responsiveness related change should also be reverted, too.

Reply:
@PeterDaveHello Just the extra linebreaks I've added?